.travis/script.sh

```diff
@@ -39,11 +39,19 @@ case "${BUILD_TARGET}" in
         ;;
 
     linux-auto-gcc-check-happy)
+        mkdir -p $TRAVIS_BUILD_DIR/happy-test-logs/${BUILD_TARGET}/from-tmp
```

Comment:
It looks like this block and the one at 44 are identical outside of USE_LWIP=1. Please make a subroutine that both of these case statements invoke with the appropriate parameterization / arguments.

An diff against the existing code base for this might look like:

```
diff --git a/.travis/script.sh b/.travis/script.sh
index 1624b492..a002f28d 100755
--- a/.travis/script.sh
+++ b/.travis/script.sh
@@ -28,6 +28,20 @@ die()
     exit 1
 }
 
+#
+# do_linux_gcc_check_happy <use_lwip>
+#
+# Run a Linux/GCC using the standalone wrapper makefile with code 
+# coverage, debugging enabled, and functional network tests for either 
+# BSD sockets or LWIP, depending on the value of 'use_lwip'.
+#
+do_linux_gcc_check_happy()
+{
+    use_lwip=${1}
+
+    sudo make -f Makefile-Standalone DEBUG=1 TIMESTAMP=1 COVERAGE=1 USE_LWIP=${use_lwip} BuildJobs=24 check
+}
+
 case "${BUILD_TARGET}" in
 
     linux-auto-clang)
@@ -39,11 +53,11 @@ case "${BUILD_TARGET}" in
         ;;
 
     linux-auto-gcc-check-happy)
-        sudo make -f Makefile-Standalone DEBUG=1 TIMESTAMP=1 COVERAGE=1 BuildJobs=24 check
+        do_linux_gcc_check_happy 0
         ;;
 
     linux-lwip-gcc-check-happy)
-        sudo make -f Makefile-Standalone DEBUG=1 TIMESTAMP=1 COVERAGE=1 USE_LWIP=1 BuildJobs=24 check
+        do_linux_gcc_check_happy 1
         ;;
 
     linux-lwip-clang)
```

Reply:
yup, working on it now